src/examples/java_monorepo_workflows.yml

```diff
@@ -16,7 +16,7 @@ usage:
 
   orbs:
     compliance: ricardo/compliance-orb@2
-    ric-orb: ricardo/ric-orb@2.0
+    ric-orb: ricardo/ric-orb@2
```

Comment:
What is the reason behind this change?
The orbs are explicitely versioned using [SemVer](https://semver.org/), so it should look like @2.0.0 instead of @2

Reply:
Alright, makes sense to me 👍 
Thanks for the explanation!